src/main/java/com/thebois/models/Position.java

```diff
@@ -42,10 +42,10 @@ public float distanceTo(final Position other) {
     /**
      * Calculates the euclidean distance between two positions.
      *
-     * @param otherPosX The x-coordinate to compare distance to
-     * @param otherPosY The y-coordinate to compare distance to
+     * @param otherPosX The x-coordinate to compare distance to.
+     * @param otherPosY The y-coordinate to compare distance to.
      *
-     * @return The distance between the points
+     * @return The distance between the points.
```

Comment:
Being punctual I see 👍

Reply:
Btw you should accept/request changes instead of commenting, to actually affect the status of the PR!

Reply:
Thanks for the pro-tip